output-facebookmessenger/src/models/FacebookMessengerResponse.ts

```diff
@@ -82,4 +83,8 @@ export class FacebookMessengerResponse {
   @IsOptional()
   @IsEnum(MessageTag)
   tag?: MessageTag;
+
+  hasSessionEnded(): boolean {
+    return true;
```

Comment:
This should probably be the same value as `hasSessionEnded` in `GoogleBusinessResponse`, because these platforms should behave as similar as possible.

Reply:
Makes sense :+1: